include/rfl/Processors.hpp

```diff
@@ -38,6 +39,10 @@ struct Processors<Head, Tail...> {
       std::disjunction_v<internal::is_add_tags_to_variants<Head>,
                          internal::is_add_tags_to_variants<Tail>...>;
 
+  static constexpr bool add_namespaced_tags_to_variants_ =
```

Comment:
FYI, at one point I considered setting both `add_tags_to_variants_` and `is_add_namespaced_tags_to_variants` for the new `rfl::AddNamespacedTagsToVariants` to try to simplify some shared code-paths, but decided against that, to keep it distinct from the other processors.